deployments/liqo/Chart.yaml

```diff
@@ -21,4 +21,8 @@ version: "0.1"
 # AppVersion is commented by default. Uncomment it or add it as extra-args to helm package if you want to release a new Liqo version
 # appVersion:
 
-dependencies: []
+dependencies:
+- name: capsule
+  version: v0.1.0-rc2-liqo
+  repository: http://helm.liqo.io/capsule
```

Comment:
Would add a comment that this is a temporary solution waiting for the new version of capsule to be released.